### ISSUE TITLE
When converting the weight file to an onnx file, there are two identity nodes participating in the BN layer. Have you encountered this problem

### ISSUE THREAD

**Issue by Daniu-jiao** [XuJiacong/PIDNet#98]
This mistake in class PAPPM last shortcut moduel.

out = self.compression(torch.cat(x_list, 1)) + self.shortcut(x)

self.shortcut = nn.Sequential(
                                            BatchNorm(inplanes, momentum=bn_mom),
                                            nn.ReLU(inplace=True),
                                            nn.Conv2d(inplanes, outplanes, kernel_size=1, bias=False), )
